src/main/java/seedu/address/logic/commands/appointmentcommands/EditAppointmentCommand.java

```diff
@@ -129,7 +133,7 @@ public boolean equals(Object other) {
      * corresponding field value of the appointment.
      */
     public static class EditAppointmentDescriptor {
-        private Email email;
+        private Name name;
         private SubjectName subjectName;
         private AppointmentDateTime dateTime;
```

Comment:
I think we might need another `dateTime` variable here because from what I see, the tutee can only edit either the `timeFrom` or `timeTo`. If we update one of them, the another's value will be overridden due to how `updatedTimeFrom` and `updatedTimeTo` is instantiated. Maybe @kingsleykuan can double confirm on this? Thanks! The rest of the codes LGTM!

Reply:
True haha. Will make the changes.